### ISSUE TITLE
Blob NavBar Menu

### ISSUE THREAD

**Issue by nis6** [nis6/Reader-s-Attic#5]
###  Warning: validateDOMNesting(...): ```<tr>``` cannot appear as a child of ```<table>```

Just do what the warning suggests:
```html
<table style={tableStyle}>
   <tbody>
       <tr style={rowStyle}>
           <td style={leftColumnStyle}>Battery</td>
           <td><StatsBar/></td>
        </tr>
   </tbody>
</table>
```
Browsers need the <tbody> tag. If it is not in your code, then the browser will automatically insert it. This will work fine on first render, but when the table gets updated, then the DOM tree is different from what React expects. This can give strange bugs, therefore React warns you to insert the <tbody>. It is a really helpful warning.

[Refer to ISSUE](https://github.com/facebook/react/issues/5652)

**Comment by nis6** [comment]
The blob image causing issues, removing the image all links work
![image](https://user-images.githubusercontent.com/47380034/151542179-1d666459-3bd8-4303-b21e-5247becc1bfa.png)
```js
<img
              src={home_illustrations.blob}
              alt="blob"
              style={{
                width: "140%",
                position: "absolute",
              }}
            />
```
![image](https://user-images.githubusercontent.com/47380034/151541811-e7064a05-b4ac-4cab-806a-17948a67f030.png)

Adding **zIndex:-1**  to style shifts the image 
![image](https://user-images.githubusercontent.com/47380034/151542381-1f82944d-b152-4b46-8e07-4c942ee5f38c.png)

**Comment by nis6** [comment]
For above Comment: 

- With z-index, blob image is not shifted it is sent back in stacking order and NavbarContainer having bg color as main color overlaps the upper part of the blob image.
- Links weren't working because the menu was overlapped with Main text container most of the time, also z-index might also have been a contributing factor to this (because on search page there is no text container to overlap)

**Comment by nis6** [comment]
- MenuLinks with styled.a was responding to cursor but not redirecting to respective pages, apparently styled(Link) fixes the issue here, but this fix doesn't work for social links issue #7
